src/create-handler-map.ts

```diff
@@ -26,7 +26,47 @@ export type CreateHandlerMap<TPrevState> = <
 >(
   actionCreators: TActionCreator | TActionCreator[],
   handler: Handler<TPrevState, TAction, TNextState>
-) => HandlerMap<TPrevState, TAction, TNextState>
+) => HandlerMap<TPrevState, TAction, TNextState>) & {
+  default: <
+    TActionCreator extends ActionCreator<any>,
+    TNextState extends TPrevState,
+    TAction extends AnyAction = TActionCreator extends (...args: any[]) => infer T
+      ? T
+      : never
+    >(
+    handler: Handler<TPrevState, TAction, TNextState>
+  ) => { default: Handler<TPrevState, TAction, TNextState> }
+}
+
+function handle<
+  TActionCreator extends ActionCreator<any>,
+  TPrevState,
+  TNextState extends TPrevState,
+  TAction extends AnyAction = TActionCreator extends (...args: any[]) => infer T
+    ? T
+    : never
+  >(
+  actionCreators: TActionCreator | TActionCreator[],
+  handler: Handler<TPrevState, TAction, TNextState>
+) {
+  return (Array.isArray(actionCreators) ? actionCreators : [actionCreators])
+    .map(getType)
+    .reduce<HandlerMap<TPrevState, TAction, TNextState>>((acc, type) => {
+      acc[type] = handler
+      return acc
+    }, {} as any)
+}
+
+handle.default = <
+  TActionCreator extends ActionCreator<any>,
+  TPrevState,
+  TNextState extends TPrevState,
+  TAction extends AnyAction = TActionCreator extends (...args: any[]) => infer T
+    ? T
+    : never
+  >(
+  handler: Handler<TPrevState, TAction, TNextState>
+) => ({ default: handler })
```

Comment:
Please use a symbol for `default` handler. Otherwise it may cause a breaking change.